<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HandleBom</class>
 <widget class="QMainWindow" name="HandleBom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>现状BOM</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="tableView">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_1">
          <item>
           <widget class="QPushButton" name="PushButton">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>提交修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ReturnButton">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>撤销修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="AddButton">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>添加记录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DeleteButton">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>删除选中行</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLineEdit" name="LineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="FindButton">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>查找厂内模号</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>理论BOM</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="tableView_1">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="PushButton_1">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>提交修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ReturnButton_1">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>撤销修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="AddButton_1">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>添加记录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DeleteButton_1">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>删除选中行</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLineEdit" name="LineEdit_1"/>
          </item>
          <item>
           <widget class="QPushButton" name="FindButton_1">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>查找厂内模号</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>废料价格</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="tableView_2">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="PushButton_2">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>提交修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ReturnButton_2">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>撤销修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="AddButton_2">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>添加记录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DeleteButton_2">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>删除选中行</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLineEdit" name="LineEdit_2"/>
          </item>
          <item>
           <widget class="QPushButton" name="FindButton_2">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>查找存货名称</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>纸箱、吸塑盒单价</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableView" name="tableView_3">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="PushButton_3">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>提交修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ReturnButton_3">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>撤销修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="AddButton_3">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>添加记录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DeleteButton_3">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>删除选中行</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLineEdit" name="LineEdit_3"/>
          </item>
          <item>
           <widget class="QPushButton" name="FindButton_3">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>查找厂内模号</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>工时</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTableView" name="tableView_4">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="PushButton_4">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>提交修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ReturnButton_4">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>撤销修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="AddButton_4">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>添加记录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DeleteButton_4">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>删除选中行</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLineEdit" name="LineEdit_4"/>
          </item>
          <item>
           <widget class="QPushButton" name="FindButton_4">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>查找产品料号</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>机台成本</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableView" name="tableView_5">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QPushButton" name="PushButton_5">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>提交修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ReturnButton_5">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>撤销修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="AddButton_5">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>添加记录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DeleteButton_5">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>删除选中行</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLineEdit" name="LineEdit_5"/>
          </item>
          <item>
           <widget class="QPushButton" name="FindButton_5">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>查找产品料号</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>PushButton</tabstop>
  <tabstop>ReturnButton</tabstop>
  <tabstop>AddButton</tabstop>
  <tabstop>DeleteButton</tabstop>
  <tabstop>LineEdit</tabstop>
  <tabstop>FindButton</tabstop>
  <tabstop>tableView_1</tabstop>
  <tabstop>PushButton_1</tabstop>
  <tabstop>ReturnButton_1</tabstop>
  <tabstop>AddButton_1</tabstop>
  <tabstop>DeleteButton_1</tabstop>
  <tabstop>LineEdit_1</tabstop>
  <tabstop>FindButton_1</tabstop>
  <tabstop>tableView_2</tabstop>
  <tabstop>PushButton_2</tabstop>
  <tabstop>ReturnButton_2</tabstop>
  <tabstop>AddButton_2</tabstop>
  <tabstop>DeleteButton_2</tabstop>
  <tabstop>LineEdit_2</tabstop>
  <tabstop>FindButton_2</tabstop>
  <tabstop>tableView_3</tabstop>
  <tabstop>PushButton_3</tabstop>
  <tabstop>ReturnButton_3</tabstop>
  <tabstop>AddButton_3</tabstop>
  <tabstop>DeleteButton_3</tabstop>
  <tabstop>LineEdit_3</tabstop>
  <tabstop>FindButton_3</tabstop>
  <tabstop>tableView_4</tabstop>
  <tabstop>PushButton_4</tabstop>
  <tabstop>ReturnButton_4</tabstop>
  <tabstop>AddButton_4</tabstop>
  <tabstop>DeleteButton_4</tabstop>
  <tabstop>LineEdit_4</tabstop>
  <tabstop>FindButton_4</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
